build.assets/install

```diff
@@ -33,7 +33,7 @@ echo "Thanks for installing Teleport."
 echo ""
 echo "Is it OK if we collect some info about your install?"
 echo "Please run this command to send in a survey."
-echo "optional: Replace email to join our newsletter and get a swag package."
-echo "$ curl -X POST https://usage.teleport.dev -F OS=${machine} -F use-case="access my ..." -F email="alice@example.com"
+echo "Optional: Replace email to join our newsletter and get a swag package."
+echo "$ curl -X POST https://usage.teleport.dev -F OS=${machine} -F use-case=\"access my ...\" -F email="alice@example.com""
```

Comment:
```suggestion
echo "$ curl -X POST https://usage.teleport.dev -F OS=${machine} -F use-case=\"access my ...\" -F email=\"alice@example.com\""
```

Reply:
Any reason to add quotes for Email?  Since I won't expect spaces in email, they won't need to be escaped. See the above example output?

Reply:
In general when doing anything with shells I just quote all the things in case someone has overridden their `IFS` to something weird or they're not using a standard shell.

If you don't want to use quotes then I'd remove them completely - at the moment you've got `echo "..... -F email="blah""` so two sets of quotes without any escaping, it should be `echo "..... -F email=blah"`